packages/blade/src/components/Checkbox/_decisions/decisions.md

```diff
@@ -0,0 +1,220 @@
+# Checkbox Decisions <!-- omit in toc -->
```

Comment:
image ko render to krdo bhai markdown me, can't find it 🙈

Reply:
Oh that Image is for "component architecture" I added that section initially in the decision file but to moved it to the PR later. https://github.com/razorpay/blade/pull/544#issue-1301661653

Reply:
didn't get you? how can someone look at this image? apart from navigating directly from the directory?

Reply:
They won't cuz I'll delete it 🤣 There's no reference to this image anymore in the Decision file. 

We had before when [we had the "component architecture" section.](https://github.com/razorpay/blade/blob/91c15c6bd2ce889f4ec866499dc028e0fd22120c/packages/blade/src/components/Checkbox/_decisions/decisions.md#component-architecture)